translations/monero-core_de.ts

```diff
@@ -140,19 +140,19 @@
     <message>
         <location filename="../components/DaemonConsole.qml" line="108"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Schließen</translation>
     </message>
     <message>
         <location filename="../components/DaemonConsole.qml" line="119"/>
         <source>command + enter (e.g help)</source>
-        <translation type="unfinished"></translation>
+        <translation>Befehl + Enter (z.B. help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <source>Daemon doesn&apos;t appear to be running</source>
-        <translation type="vanished">Daemon scheint nicht aktiv zu sein</translation>
+        <translation type="vanished">Dämon scheint nicht aktiv zu sein</translation>
```

Comment:
The fact, that maybe only few non-techies use Monero _(what is not proven - but if it was, maybe it's because using Monero is too technically challenging?)_, is not a valid argument to leave it at the techie-style language. Actually, we could leave everything in English as a consequence, why bother with a translation at all? Tell me, why do they build barrier-free entries to buildings, although no wheelchair driver ever used the steps before? True, by that I exaggerate pointedly... ;)

I personally know of a flight attendant and she bought some Monero last week after she got it all explained from her boyfriend. Of course, she's clever _(come on, she owns some Monero, that alone makes you a lot smarter!)_ and wouldn't suspect an evil spirit in her laptop, when working with the GUI. But with something like "Service" or "Dienst" she would at least have expectations what the program/function does, without having to google it first. It's more respectful not to expect every user to be not less than an advanced amateur.

The French use "démon" and not an English word, what's maybe not too surprising for them. Whereas the Spanish use "daemon" and not their native-language "demonio". 

**So whatever we use, we're among good friends! :)**